docs/kb/serve.md

```diff
@@ -62,7 +62,7 @@ Reset the state only once. Use this flag to resume a failed reset or to initiali
 
 Reset state on every file change. Do not import state and turn off state persistence.
 
-`--rebuild-proto-once` use with `--reset-once`
+`starport generate vuex`
```

Comment:
```suggestion
```

I think we should just remove this from the `serve` command doc. It's a separate command now.